setup.cfg

```diff
@@ -5,3 +5,4 @@ universal = 1
 select =
 ignore = E226,E24
 max_line_length = 79
+max_doc_length = 79
```

Comment:
shouldn't this be 72 ?

Reply:
Similar to the tox.ini, I seem to recall setting this to 72 will cause it to fail on the current pycodestyle codebase.